src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+class BasicValidationServiceTest {
```

Comment:
Add a happy path to all tests below with assertDoesNotThrow

Reply:
done